### ISSUE TITLE
change the webpage color theme

### ISSUE THREAD

**Issue by Arman-ali-khan-786** [Arman-ali-khan-786/CardUI#41]

**Comment by abitsalihu** [comment]
Hey @Arman-ali-khan-786, I would like to help with this issue. :)

**Comment by Arman-ali-khan-786** [comment]
> Hey @Arman-ali-khan-786, I would like to help with this issue. :)

please wait for HacktoberFest to start bcz this repo is participating in it ok

**Comment by abitsalihu** [comment]
Oh okay, I'll work on this tomorrow after it starts then?

**Comment by Nzarite** [comment]
Hey @Arman-ali-khan-786, I would like to work on this issue.

**Comment by sudip004** [comment]
can i work on it

**Comment by abitsalihu** [comment]
@Arman-ali-khan-786 Should I start working on this?

**Comment by Arman-ali-khan-786** [comment]
> @Arman-ali-khan-786 Should I start working on this?

yes

**Comment by Arman-ali-khan-786** [comment]
> Hey @Arman-ali-khan-786, I would like to work on this issue.

@abitsalihu is working on it

**Comment by Arman-ali-khan-786** [comment]
> can i work on it

@abitsalihu  is working on it hold on for now

**Comment by himanshu251001** [comment]
@Arman-ali-khan-786 assign me

**Comment by Arman-ali-khan-786** [comment]
@abitsalihu r u still on it??

**Comment by abitsalihu** [comment]
@Arman-ali-khan-786 yes I am, I will do the pr later today as I had some laptop problems. 🙂

**Comment by Arman-ali-khan-786** [comment]
> @Arman-ali-khan-786 yes I am, I will do the pr later today as I had some laptop problems. 🙂

well i haven't got any pr

**Comment by abitsalihu** [comment]
@Arman-ali-khan-786 yes you did, but you get accepting other pull requests over mine, and then you wanted me to do pull requests and push all the time, I did what you wanted then you insulted me by saying I didn't review my code (which I didn't have to do cuz you never asked for me to review it all you asked was to make a pull request and push it again), since something else was the problem when you never mentioned it. So I never sent a new PR cuz you don't know how to handle a project. 

Have a great day!

**Comment by Arman-ali-khan-786** [comment]
> @Arman-ali-khan-786 yes you did, but you get accepting other pull requests over mine, and then you wanted me to do pull requests and push all the time, I did what you wanted then you insulted me by saying I didn't review my code (which I didn't have to do cuz you never asked for me to review it all you asked was to make a pull request and push it again), since something else was the problem when you never mentioned it. So I never sent a new PR cuz you don't know how to handle a project.
> 
> Have a great day!

You were not checking the recent pr's but now you are complaining about when it's your fault, maybe be you should learn how to contribute first then tell me about how to handle projects and all that stuffs..
- Learn the basics and have some ethics   Have a nice day !

**Comment by Gautam-666** [comment]
I would like to work on it ....do you have some theme or design in your mind which you want me to implement or should I work on it as per my wish...

**Comment by Arman-ali-khan-786** [comment]
> I would like to work on it ....do you have some theme or design in your mind which you want me to implement or should I work on it as per my wish...

you come up with your own, Ok....

**Comment by panditsamik** [comment]
Can I work on this issue #41 for Hackodex 2023? @Arman-ali-khan-786

**Comment by Arman-ali-khan-786** [comment]
> Can I work on this issue #41 for Hackodex 2023? @Arman-ali-khan-786

@panditsamik  gautam is already working on it if he doesn't show up then you will be assigned ok

**Comment by Gautam-666** [comment]
Maybe you can close this one....it's done

**Comment by Arman-ali-khan-786** [comment]
> Maybe you can close this one....it's done

there always a scope for better theme so it will be open okk

**Comment by Arman-ali-khan-786** [comment]
> Can I work on this issue #41 for Hackodex 2023? @Arman-ali-khan-786

if you have better theme than go ahead ..

**Comment by Arman-ali-khan-786** [comment]
@panditsamik are you still on it??

**Comment by Naman73598** [comment]
@Arman-ali-khan-786 i would like to work on this issue #41 for hacktoberfest2023.

**Comment by RitvikRaj0810** [comment]
if you're still facing any issue then please assign me.

**Comment by Arman-ali-khan-786** [comment]
> if you're still facing any issue then please assign me.

@RitvikRaj0810 @Naman73598 is still working on it, if he is not able to do it then you can okk..
@Naman73598 any updates ???

**Comment by saanvibk** [comment]
Hey @Arman-ali-khan-786, I would like to work on this issue.

**Comment by Arman-ali-khan-786** [comment]
Since @Naman73598 is not responding ... then @RitvikRaj0810 you can work on this issue ok.. and @saanvibk i assign issue on first come basis so if @RitvikRaj0810 can't do it then you can work on it ok.

**Comment by saanvibk** [comment]
Ok
